xl2times/transforms.py

```diff
@@ -57,7 +57,7 @@ def remove_comment_rows(
     result = []
 
     for table in tables:
-        tag = datatypes.Tag(table.tag.split(":")[0])
+        tag = datatypes.Tag(table.tag)
```

Comment:
Random nitpickey suggestion : this file would be more readable if we did `from .datatypes import Tag` and changed all these to just `Tag(table.tag)`. 
Unless there's a good reason not to (like if there was another Tag class somewhere that needed distinguishing from)

Same with `datatype.Config, EmbeddedXlTable and TimesModel` references.

Reply:
I think this would make a lot of sense. If @siddharth-krishna agrees we could change all them in a separate PR?

Reply:
Nice idea, thanks, I opened an issue to track it: https://github.com/etsap-TIMES/xl2times/issues/220